### ISSUE TITLE
Iris nodules

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#9663]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
Wiki eye: **Fuchs’ Heterochromic Iridocyclitis** (FHI), also known as Fuchs’ uveitis syndrome (FUS), was first described in 1906 by Austrian ophthalmologist, Ernst Fuchs, who reported a series of 38 patients with iris heterochromia, cyclitis, and cataract. Since then, FHI has been further characterized as a constellation of clinical findings, which classically includes low-grade unilateral anterior inflammation with stellate keratic precipitates (KP), iris heterochromia, and prominent iris and angle vessels that bleed after paracentesis (Amsler sign)
(........) Iris nodules, particularly Koeppe and Busacca nodules, can be seen in FHI and may initially lead to a misdiagnosis of granulomatous uveitis.
(https://eyewiki.org/Fuchs_Heterochromic_Iridocyclitis)

PMID: 27781126: The most common clinical findings were small to medium-sized keratic precipitates (95.6% of the eyes).
According to SUN criteria [21] no to moderate inflammation in anterior chamber (cells from 0 to ≤2+) was observed in 158 eyes (99.37%), and 1 eye only presented 3+ cells (0.6%). Sixty eyes (37.7%) presented **iris nodules**: Koeppe nodules in 48 cases (30.2%), Busacca in 2 (1.2%), and both types in 10 eyes (6.3%).

 PMID: 16458965: The most common symptom was blurred or decreased vision (86%). Stellate and medium-sized keratic precipitates (KPs) were noted in 108 eyes (91.5%). A mild anterior chamber (AC) reaction was observed in all the affected eyes. Heterochromia was observed in only 15 affected eyes, although there were varying degrees of iris depigmentation in all patients. **Iris nodules,** mostly Koeppe, were present in 28.0% of the affected eyes

PMID: 17986291 **Iris nodules** can be seen mostly on the pupillary margin (Koeppe) but in some cases even on the iris stroma (Busacca)

11815885[PMID]: **Iris nodules**, previously described as a frequently occurring sign in FHI, were present in only two eyes. In our series, they were not associated with a worse prognosis

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Fuchs heterochromic iridocyclitis ORPHA:263479

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@carolina-f 

adding 
HP:0034820  
Koeppe nodules

**Comment by pnrobinson** [comment]
added
Busacca nodules
HP:0034821
